topdown/topdown.go

```
@@ -697,6 +697,16 @@ func ValueToFloat64(v ast.Value, ctx *Context) (float64, error) {
 	return f, nil
 }
 
+// ValueToInt returns the underlying Go value associated with an AST value.
+// If the value is a reference, the reference is fetched from storage.
+func ValueToInt(v ast.Value, ctx *Context) (int, error) {
+	x, err := ValueToFloat64(v, ctx)
+	if err != nil {
+		return 0, err
+	}
+	return int(x), nil
```

Probably want to return an error if x is not round.